extensions/launch-agents/components/LaunchAgentDetails.tsx

```diff
@@ -77,7 +77,7 @@ export default function LaunchAgentDetails({
 
   const deleteFile = () => {
     try {
-      execSync(`rm ${selectedFile}`);
+      execSync(`rm -rf ${selectedFile}`);
```

Comment:
What about using the built-in feature [trash](https://developers.raycast.com/api-reference/utilities#trash) here

Reply:
but that only accepts a file right? this change adapts so that it can also has a folder for some reason.

Reply:
We can do it later on for sure 🙂